modMcf/src/org/aion/mcf/config/CfgLog.java

```diff
@@ -85,6 +99,21 @@ public String toXML() {
             xmlWriter.writeCharacters("\r\n\t");
             xmlWriter.writeStartElement("log");
             xmlWriter.writeCharacters("\r\n");
+
+            /** XML - Displays tag/entry in the config.xml */
+            xmlWriter.writeCharacters("\t\t");
+            xmlWriter.writeStartElement("log-file");
+            xmlWriter.writeCharacters(this.logFile + "");
+            xmlWriter.writeEndElement();
+            xmlWriter.writeCharacters("\r\n");
+
+            /** XML - Displays log-path in the config.xml */
+            xmlWriter.writeCharacters("\t\t");
+            xmlWriter.writeStartElement("log-path");
```

Comment:
could you add comments here regarding the meaning of the configuration (similar to the ones in the `api` and `db`)

Reply:
Added comments into the config.xml file